### PR TITLE
Include uboot bits in installer image

### LINKED ISSUE #4288: arm64 installer issue: files are missing at the EFI partition of the destination storage device
### Description

When installing EVE on an ARM device, typically the following files are expected to be on the EFI partition:

```
bcm2711-rpi-4-b.dtb
bcm2711-rpi-cm4.dtb
boot
config.txt
EFI
fixup4.dat
fr201.txt
overlays
start4.elf
startup.nsh
u-boot.bin
ubootefi.var
UsbInvocationScript.txt
```

However, only the following files are available after the installation:

```
cmdline
dtb
EFI
ubootefi.var
```

This issue causes RPi based devices to fail to boot after installation.

### How to reproduce

1. Generate a installer-raw image
2. Flash to a USB Stick
3. Try to install on a RPi based device

### Affected versions

* EVE-OS version: tested on master branch
* EVE-OS hardware architecture: only arm64 is affected

### Additional context

Thanks @dpoulios to first observe this issue on a RPi based device.

### PR DESCRIPTION
**Pull request by deitch**
Fixes #4288

They already are added to the installer ESP, but the installer itself did not have access to them, and thus would not install them on the target disk.

The source is `pkg/u-boot`.

This PR adds the contents of that pkg to `installer.yml` as a volume, and then mounts it into the installer container, which makes it available to `make-raw` as `/parts/boot`, which then knows what to do with.

There is a bit of duplication here, but this is cleaner than alternatives.

At some point we should eliminate the `installer/boot` directory and make it all use what is in the `installer.tar`. That needs some changes to `rootfs.tar`, which would be beyond scope of this PR.

cc @rene

### CODE REVIEW

**Review by rene — APPROVED** [review]
LGTM

File: images/installer.yml.in [hunk 1]
```diff
@@ -29,6 +30,9 @@ volumes:
 - name: mkimage-raw-efi
   image: MKRAW_TAG
   readonly: true
+- name: uboot
```
Comment (OhmSpectator) [inline images/installer.yml.in:33]: Isn't it ARM specific? Shouldn't we add it conditionally somehow?

**Comment by deitch** [comment]
pull rate limits...

File: images/installer.yml.in (hunk 1 above)
Comment (deitch) [reply images/installer.yml.in:33]: Nah, it is available for all platforms, even riscv64. Different content. It works on all of them.

**Review by OhmSpectator — APPROVED** [review]

File: images/installer.yml.in (hunk 1 above)
Comment (OhmSpectator) [reply images/installer.yml.in:33]: Strange... Why do we need u-boot for x86, for example... But ok, let it be...

**Pull request merged**